routes/web.php

```diff
@@ -348,27 +349,29 @@ function (): void {
                             ->name('manager.jobs.applications');
 
                         /* View Application */
-                        Route::get('applications/{application}', 'ApplicationController@show')
+                        Route::get('jobs/{jobPoster}/applications/{application}', 'ApplicationController@show')
+                            ->middleware('can:manage,jobPoster')
                             ->middleware('can:view,application')
                             ->name('manager.applications.show');
 
                         /* View Applicant Profile */
-                        Route::get('applicants/{applicant}', 'ApplicantProfileController@show')
+                        Route::get('jobs/{jobPoster}/applicants/{applicant}', 'ApplicantProfileController@show')
+                            ->middleware('can:manage,jobPoster')
```

Comment:
I was unsure about this, since the jobPoster id isn't technically necessary, but after talking to the designers I see why this would make more sense to users.